index.js

```diff
@@ -61,10 +63,10 @@ class APIClient {
         return response;
     }
 
-    async publish(target = 'default', token = this.fetchToken()) {
+    async publish(target = 'default', token = this.fetchToken(), deployPercentage = undefined) {
```

Comment:
Same goes here, this can be changed to a single object: `{target, token, deployPercentage}`

I'll release a breaking version after this

Reply:
Thanks, I'm preparing a new commit for this.